templates/registration/login.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+
+{% block content%}
+    <h2>LogIn</h2>
```

Comment:
add error message to page when there are errors in form

Reply:
I did add